app-emulation/xenserver-pv-version/xenserver-pv-version-6.2.0.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI=""
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64"
+KEYWORDS="amd64 arm64"
```

Comment:
Only applicable to XenServer deployments, unlikely to be useful on ARM I assume. Add a conditional depend to the coreos ebuild instead.